src/tattoo.ts

```diff
@@ -79,6 +80,26 @@ const cli = cliArgs([
         "Set to clone all repos from remote instead of updating local copies."
   },
   {
+    name: "wctflags",
+    type: String,
+    defaultValue: "-b chrome",
```

Comment:
can we add a wct flags section to the config file? I image someone preparing a tattoo test folder and running over and over...